virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -190,73 +197,71 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 ** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
-//CNV-34198 (BZ 2237287)
-* If you use Portworx as your storage solution on AWS and create a VM disk image, the created image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2237287[*BZ#2237287*])
-** As a workaround, you can manually expand the Persistent Volume Claim (PVC) to increase the available space after the initial provisioning process completes.
-
-//BZ2216038 from Ying
-* If you simultaneously clone more than 1000 VMs using the provided DataSources in the `openshift-virtualization-os-images` namespace, it is possible that not all of the VMs will move to a running state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2216038[*BZ#2216038*])
-** As a workaround, deploy VMs in smaller batches.
 
-//BZ2247593 from Ying
-* Live migration cannot be enabled for a virtual machine instance (VMI) after a hotplug volume has been added and removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2247593*])
 
 [discrete]
 [id="virt-4-15-ki-virtualization"]
 ==== Virtualization
-//4.14 Leave in per Stu
-* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+//New for 4.15
+* CPU hot unplug for Windows VMs is not supported and results in an error. 
+// Cannot link to issues with security level "Red Hat Employee" (link:https://issues.redhat.com/browse/CNV-37346[*CNV-37346*])
+
+* When adding a virtual Trusted Platform Module (vTPM) device to a Windows VM, the BitLocker Drive Encryption system check passes even if the vTPM device is not persistent. This is because a vTPM device that is not persistent stores and recovers encryption keys using ephemeral storage for the lifetime of the `virt-launcher` pod. When the VM migrates or is shut down and restarts, the vTPM data is lost. (link:https://issues.redhat.com/browse/CNV-36448[*CNV-36448*])
+
+
+//Looks like this is still unresolved for 4.15
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://issues.redhat.com/browse/CNV-33835[*CNV-33835*])
 ** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
-// Leave in for 4.14 per Stu; remove from 4.15 RNs and move content to CNV docs, per Avital
+//4.15 Check with Stu if this needs to be moved to CNV docs
```

Comment:
@stu-gott Do we still need to keep the below known issue in the release notes?

Reply:
The release note still applies. We are currently working toward virtiofs support in KubeVirt, which is the precursor to addressing this. Rough ETA for fixing this issue is forecast as 4.18+. If it's a problem to leave a RN in for that long we could consider moving this to somewhere in the documentation, but we do plan to fix it once we're able.

Reply:
Thanks for the update. We will leave the RN as is.